examples/hello_world/lib/arabic.dart

```diff
@@ -1,7 +1,5 @@
 // Copyright 2014 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
```

Comment:
@AdamMusa You have to keep the blank line in line 4 and line 6. Just please don't add an additional blank line to the end of the file.